src/main/kotlin/app/hashers/CommitCrawler.kt

```diff
@@ -248,4 +269,34 @@ object CommitCrawler {
             listOf()
         }
     }
+
+    /**
+     * Return a list of paths that should be ignored in commit analysis.
+     */
+    private fun getIgnoredPaths(repo: Repository): List<String> {
+        return try {
+            val list = mutableListOf<String>()
+            val file = File(repo.workTree, CONF_FILE_PATH)
```

Comment:
We discussed that file structure and paths to vendor may change in time as well as config file. So it makes sense to get config from the currently processed commit. Suggested solution:
```
            var conf = null
            val treeWalk = TreeWalk(repo)
            treeWalk.reset(commitId)
            treeWalk.setFilter(PathFilter.create("./.sourcerer-conf"))
            if (treeWalk.next()) {
                val obj = repo.open(objectId)
                conf = RawText(obj.bytes)
            }
```

Reply:
oh, right, will fix it